views/songlist.jade

```diff
@@ -1,16 +1,23 @@
 extends layout
 
 block content
-    include topnav
-    h1 Song List
+    nav.navbar.navbar-transparent
+        .container-fluid
+            .navbar-header
+            <a class='navbar-brand' href="/" style="color: black"> <span class="glyphicon glyphicon-home" aria-hidden="true"></span> SPF.IO Songbase </a>
+            .collapse.navbar-collapse
+                include topnav-user
+    h1 Songs
+    #searchBox
```

Comment:
consider keeping your ids consistent in  style. 
here you have #searchBox later you have #song-table.
Sticking to one convention is helpful.